.circleci/src/workflows/build-and-test-all.yml

```diff
@@ -134,13 +135,31 @@
             - 'OFF'
             base-image:
             - amazonlinux:2023.3.20240219.0
```

Comment:
For my own education, is there a set of base images that we aim to support? And if Amazon Linux is one of them, is that the only reason we are still testing NGINX 1.24, which would otherwise be out of support?

Reply:
> Amazon Linux is one of them, is that the only reason we are still testing NGINX 1.24, which would otherwise be out of support?
yes.